src/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 900;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.search-form {
+  margin-top: 20px;
+  position: relative;
+  height: 70px;
+}
+
+.search-form-small {
+  height: 42px;
+}
+
+.search-input {
+  width: 100%;
+  height: 100%;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.search-input::placeholder {
+  color: #3d4e61;
+}
+
```

Comment:
When you have focus state your placeholder should not change, but here it is getting bolder